.github/workflows/deploy.yml

```diff
@@ -195,5 +197,47 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - name: TODO - Documentation Deploy Steps
-        run: echo "TODO - Documentation Deploy Steps"
+      - name: Documentation Deploy Steps
+        run: echo "Documentation Deploy Steps"
+
+      - name: Checkout Project
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3.7.0
+        with:
+          node-version-file: '.nvmrc'
+          registry-url: 'https://registry.npmjs.org'
+          cache: 'yarn'
+
+      - name: Uncache Dependencies
+        uses: actions/cache@v3
+        with:
+          path: '**/node_modules'
+          key: node-modules-${{ hashFiles('./yarn.lock') }}
+
+      - name: Uncache Distributables
+        uses: actions/cache@v3
+        with:
+          path: '**/dist'
+          key: dist-${{ env.rid }}
+
+      - name: Synchronize Packages
```

Comment:
So we need to do all these actions again is it? The folder on actions environment doesn't have everything set up between the stages?

Reply:
+1 you dont need all the steps

Reply:
yes tested it without steps so will remove it